README.md

```diff
@@ -29,104 +29,57 @@ and 18181-4 is the [reference software](https://github.com/libjxl/libjxl).
 
 The library API, command line options, and tools in this repository are subject
 to change, however files encoded with `cjxl` conform to the JPEG XL specification
-and can be decoded with current and future `djxl` decoders or `libjxl` decoding library.
+and can be decoded with current and future `djxl` decoders or the `libjxl` decoding library.
 
-## Compilation
+## Installation
 
-For more details and other workflows see the "Advanced guide" below.
+[![Packaging status](https://repology.org/badge/tiny-repos/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![latest packaged version(s)](https://repology.org/badge/latest-versions/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![Arch](https://repology.org/badge/version-for-repo/arch/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![Debian Unstable](https://repology.org/badge/version-for-repo/debian_unstable/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![Fedora 37](https://repology.org/badge/version-for-repo/fedora_37/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![FreeBSD](https://repology.org/badge/version-for-repo/freebsd/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![Gentoo](https://repology.org/badge/version-for-repo/gentoo/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![Homebrew](https://repology.org/badge/version-for-repo/homebrew/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![KaOS](https://repology.org/badge/version-for-repo/kaos/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![Manjaro Stable](https://repology.org/badge/version-for-repo/manjaro_stable/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![MSYS2 mingw](https://repology.org/badge/version-for-repo/msys2_mingw/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![OpenMandriva Rolling](https://repology.org/badge/version-for-repo/openmandriva_rolling/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![openSUSE Tumbleweed](https://repology.org/badge/version-for-repo/opensuse_tumbleweed/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![Raspbian Testing](https://repology.org/badge/version-for-repo/raspbian_testing/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![Termux](https://repology.org/badge/version-for-repo/termux/libjxl.svg)](https://repology.org/project/libjxl/versions)
+[![Ubuntu 23.04](https://repology.org/badge/version-for-repo/ubuntu_23_04/libjxl.svg)](https://repology.org/project/libjxl/versions)
```

Comment:
Add MacPorts and vcpkg as well please?

Reply:
Done

Reply:
Err, the MacPorts line/link seems to have been truncated somehow?